### DISCUSSION TITLE
HA + mqtt2zigbee + philips Hue

### DISCUSSION THREAD

**Discussion by Rui10** [indomus/forum#1868] [HUB: Home Assistant]
Buongiorno, da qualche giorno ho migrato tutti le periferiche da Deconz a mqtt2zigbee e riprogrammato HA per utilizzarli con questa nuova modalità.
Unico devide che non riesco ad associare è una lampadina Philips HUE, c'è qualche procedura particolare?
Grazie

**Comment by snakuzzo** [comment]
https://www.zigbee2mqtt.io/devices/9290011370B.html

Guarda il touchlink reset

**Comment by Rui10** [comment]
Sicuramente sbaglio io, ma mi dice scan già in corso dopo avermi dato segnale di avvio...

AGGIORNAMENTO:
Rifatto ancora la procedura di pairing e si è configurata in un baleno... misteri!
Cmq risolto ;)
